spack/packages/dla-future/package.py

```diff
@@ -46,7 +46,7 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("umpire@6:")
 
     depends_on("pika +mpi")
-    depends_on("pika@0.6:")
+    depends_on("pika@main")
```

Comment:
To do before merging:
```suggestion
    depends_on("pika@0.7:")
```

Also, update the spack hash in the CI configuration to get pika 0.7.0.

Reply:
Yes it's planned but my PR in spack is not merged yet, will do right after

Reply:
Spack hash still needs updating.

Reply:
Sorry for the delay :/